src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html

```diff
@@ -0,0 +1,104 @@
+<!--
+## Introduction
+
+The AMPHTML Email format provides a subset of AMP components that you can use in email messages. Recipients of AMP emails can view and interact with the AMP components directly in the email message.
+
+  <p class="mb2">
+  <div class="p1 ampstart-card abe-info"><strong>Good to know:</strong> the <a href="https://validator.ampproject.org/#htmlFormat=AMP4EMAIL">AMPHTML Validator</a> features a special validation mode for AMPHTML Email messages.
+  </div>
+  </p>
+-->
+
+<!-- -->
+<!-- Doctype declaration is required. -->
+<!doctype html>
+<!-- This tells everyone that this is an AMPHTML email. `<html amp4email>` works too. -->
+<html ⚡4email>
+<!-- ## Head -->
+<!-- -->
+<head>
+  <!-- The charset definition must be the first child of the `<head>` tag. -->
+  <meta charset="utf-8">
+  <!-- The AMP runtime.-->
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!--
+    The AMPHTML Email boilerplate.
+  -->
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+  <!-- Import AMP components in the header, for example the `amp-carousel` component. -->
+  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+  <!--
+    CSS must be included in a `<style amp-custom>` tag within the header.
+  -->
+  <style amp-custom>
+    h1 {
+      color: red;
+    }
+  </style>
+</head>
+<!-- ## Body -->
+<!-- -->
+<body>
+  <!--
+    Just like in AMP HTML, in AMPHTML Email, most HTML tags can be used directly.
+  -->
+  <h1>Hello World!</h1>
+  <!--
+    Certain tags, such as the `<img>` tag, are replaced
+    with equivalent or slightly enhanced custom AMP HTML tags (see
+    [HTML Tags in the specification](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md)).
+  -->
+  <amp-img src="/img/amp.jpg" width="400" height="300"></amp-img>
```

Comment:
You'll need to put an absolute path here, the spec doesn't allow relative URLs.
https://ampbyexample.com/img/amp.jpg